test/shmemx/shmemx_team_broadcast.c

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
@jdinan Copyright needs to be added

Reply:
Resolving for now ... there is no copyright in the OpenSHMEM specification ...  Ref #901